pkg/asset/manifests/coreosbootimage.go

```diff
@@ -0,0 +1,99 @@
+package manifests
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/rhcos"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// mcoNamespace is where we target the configmap.
+	mcoNamespace = "openshift-machine-config-operator"
+	// configmapName is the name of the config map
+	configmapName = "coreos-bootimages"
+	// streamKey is the key that contains the stream metadata
+	streamKey = "stream"
+)
+
+var (
+	coreOSBootimagesCfgFilename = filepath.Join(manifestDir, "machine-config-operator-bootimages-config.yml")
+)
+
+// CoreOSBootimages generates the manifest with CoreOS stream metadata for the bootimages
+type CoreOSBootimages struct {
```

Comment:
We do not want to inject the ConfigMap to manifests that the cluster-bootstrap will create. We want to add the ConfigMap to the resources that the CVO creates. This is needed because the ConfigMap needs to be updated when the OpenShift version is upgraded.

Reply:
Nice catch - a great example of why code review is important.

However...this is a really interesting sub-topic that gets into the larger still-unanswered question of how we actually *use* this data once it's in the cluster.  See this thread https://github.com/openshift/enhancements/pull/679#discussion_r592390250

I think I vaguely had in the back of my mind that we'd defer this problem, the initial goal here just being having the data in-cluster at all.   Manual testing of machineset updates could involve just editing the configmap.

To be clear, I think you're right, we should update it by default. 

Anyways since you want this bit as a separate PR we can take the details of this there.